lib/utils/addon-utils.ts

```diff
@@ -118,7 +118,7 @@ function parseEksVersion(version: string): [string, number] {
 }
 
 
-export function conflictsWithAutoMode(minExpectedVersion: string | null) {
+export function conflictsWithAutoMode(addOn: string, minExpectedVersion: string | null) {
   // eslint-disable-next-line @typescript-eslint/ban-types
   return function(target: Object, key: string | symbol, descriptor: PropertyDescriptor) {
     const originalMethod = descriptor.value;
```

Comment:
In here you can get the addon class with 
```ts
  const addOnName = this.constructor.name;
```

This will make it more generic, and avoid a case when developer can accidentally pass the wrong name when using the decorator.